Doc/faq/programming.rst

```diff
@@ -549,12 +549,12 @@ desired effect in a number of ways.
 
       args = {'a': 'old-value', 'b': 99}
       func3(args)
-      print(args['a'], args['b'])
+      print(args['a'], args['b'])     # output: new-value 100
 
 5) Or bundle up values in a class instance::
 
       class callByRef:
```

Comment:
hm that’s an ugly name

Reply:
I change class name to `argSetter`, WDYT